new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleTraefikValidation.java

```diff
@@ -0,0 +1,201 @@
+
+    // first wait 30 seconds, then check if the traefik pod is ready.
+    with().pollDelay(30, SECONDS)
+        .and().with().pollInterval(10, SECONDS)
+        .conditionEvaluationListener(
+            condition -> logger.info(
+                "Waiting for traefik to be ready (elapsed time {0}ms, remaining time {1}ms)",
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .await().atMost(5, MINUTES)
+        .until(isTraefikReady(tfNamespace));
+  }
+
+  @Test
+  @Order(2)
+  @DisplayName("Upgrade the Traefik release")
+  public void testUpgradingTraefik() {
+
+    // create a new unique namespace for wls domain
+    logger.info("Creating an unique namespace for wls domain");
+    domainNamespace1 = assertDoesNotThrow(() -> createUniqueNamespace(),
+            "Failed to create an unique namespace due to ApiException");
+    logger.info("Created a new namespace called {0}", domainNamespace1);
+
+    // helm install parameters
+    tfHelmParams = new HelmParams()
+            .releaseName("traefik-operator")
+            .namespace(tfNamespace)
+            .chartDir("stable/traefik");
+
+    TraefikParams tfParams = new TraefikParams()
+            .helmParams(tfHelmParams)
+            .nameSpaces("{" + tfNamespace + "," + domainNamespace1 + "}");
+
+    // upgrade Traefik
+    assertThat(upgradeTraefik(tfParams))
+            .as("Test upgradeTraefik returns true")
+            .withFailMessage("upgradeTraefik() did not return true")
+            .isTrue();
+    logger.info("Traefik is upgraded in namespace {0}", tfNamespace);
+
+    // verify the value of the release 'kubernetes.namespaces' got updated
+    String helmCmd = String.format("helm get values %1s -n %2s", tfHelmParams.getReleaseName(), tfNamespace);
+    assertThat(getExpectedResult(helmCmd, domainNamespace1))
+            .as("Get the expected value")
+            .withFailMessage("Did not get the expected value")
+            .isTrue();
+  }
+
+  @Test
+  @Order(3)
+  @DisplayName("Create an ingress per domain")
+  public void testCreatingIngress() {
+    // helm install parameters for ingress
+    ingressParam = new HelmParams()
+            .releaseName("sample-domain1-ingress")
+            .namespace(domainNamespace1)
+            .chartDir(projectRoot + "/../kubernetes/samples/charts/ingress-per-domain");
+
+    assertThat(createIngress(ingressParam, "sample-domain1", "sample-domain1.org"))
+            .as("Test createIngress returns true")
+            .withFailMessage("createIngress() did not return true")
+            .isTrue();
+    logger.info("ingress is created in namespace {0}", domainNamespace1);
+
+    // check the ingress is created
+    String cmd = "kubectl get ingress -n " + domainNamespace1;
+    assertThat(getExpectedResult(cmd, "sample-domain1-traefik"))
+            .as("ingress is created")
+            .withFailMessage("ingress is not created in namespace " + domainNamespace1)
```

Comment:
include the name of the ingress, not just the namespace please

Reply:
done